### ISSUE TITLE
[Bug]: On application deployment "Application exited with an error"; no error shown in Build Logs; Application Logs hang indefinitely

### ISSUE THREAD

**Issue by spencerflagg** [coollabsio/coolify#415]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When I build and start an 11ty project (via a GitLab integration) the deployment appears to complete, the build logs have no error, and they say "Deployment successful; Proxy will be updated shortly". However, a red alert icon shows next to the play button with the hover "Application exited with an error".

When I try to check the Application Logs, the page hangs indefinitely on "Waiting for the logs..."

### Steps To Reproduce

1. Add GitLab as a source; add an 11ty application from the GitLab source
2. In the settings of the 11ty application, set up with either Coolify's default **yarn** commands, or with **npm** commands; (both fail in the same way)
3. Set the application's Publish Directory to `_site`
4. add a URL like `https://subdomain.domain.tld`
5. Save and click "build and start application"

### Version

2.6.2 & 2.7.0

**Comment by andrasbacsai** [comment]
I tried to reproduce it, but I could not. I'm using [this basic](https://github.com/coollabsio/eleventy-example) repository.

Can you please share your repository (if it's public)? Because I'm not sure if it's related to Coolify.

**Comment by spencerflagg** [comment]
My project isn't public - but this is the repository I started with:
https://github.com/ixartz/Eleventy-Starter-Boilerplate
(It's one of the major blog templates from 11ty.dev.)

The `package.json` is set up quite differently:
```
  "scripts": {
    "build-dev:webpack": "webpack",
    "watch:webpack": "webpack --watch",
    "watch:eleventy": "cross-env ELEVENTY_ENV=development eleventy --serve",
    "dev": "npm-run-all clean build-dev:webpack --parallel watch:*",
    "build:webpack": "cross-env NODE_ENV=production webpack --mode production",
    "build:eleventy": "cross-env ELEVENTY_ENV=production eleventy",
    "build": "run-s clean build:*",
    "serve:local": "serve _site",
    "serve": "run-s build serve:local",
    "clean": "rimraf _site",
    "format:js": "prettier '**/*.js' --write && eslint '**/*.js' --fix",
    "format:json": "prettier '**/*.json' --write",
    "format": "run-s format:*",
    "lint": "eslint --ext .js ."
  },
```

Admittedly, I didn't look into this issue any further when I realized that the logs weren't showing up.

**Comment by andrasbacsai** [comment]
I've deployed the boilerplate repository without modifying any configuration, and it [works](http://test.example.coolify.io/).

Can you build and serve it locally?

**Comment by spencerflagg** [comment]
Yes, I can build and serve locally.

UPDATE: I tried removing the application and re-adding it this morning, and something was different:
- All of the previous times that I'd added the application, **NodeJS** was selected by default as the build pack and I had to **manually** select 11ty. This time, 11ty was selected by default.
- All of the previous times I'd needed to add `_site` manually.
- All of the previous times, the build logs were several hundred lines long, and this time there were only 10 steps.
- This time deployment completed with no issues! :man_facepalming: 

I'm flummoxed. The repository has not been modified since I deployed originally. Perhaps it's possible it was a v2.6.2 bug and I possibly didn't reinstall the app between v2.6.2 and v2.7.0?

_Either way, thank you very much for your time! I love the product, love the vision, and will be moving several clients' sites to Coolify from clunkier deployment workflows very shortly._
